### PR TITLE
Ldap no memberof with memberuid

### PR DESCRIPTION
**Pull request by andreas-p**
As soon as an ldap server supports memberof, it is used for posixGroup as well by default, which cannot work because memberUid isn't supported (memberof requires a full dn, not just uid)

### CODE REVIEW

File: apps/user_ldap/lib/Group_LDAP.php [hunk 1]
```diff
@@ -496,9 +496,11 @@ public function getUserGroups($uid) {
 
 		// if possible, read out membership via memberOf. It's far faster than
 		// performing a search, which still is a fallback later.
+		// memberof doesn't support memberuid, so skip it here.
 		if(intval($this->access->connection->hasMemberOfFilterSupport) === 1
 			&& intval($this->access->connection->useMemberOfToDetectMembership) === 1
-		) {
+		    && strtolower($this->access->connection->ldapGroupMemberAssocAttr) !== 'memberuid'
```
Comment (blizzz) [inline apps/user_ldap/lib/Group_LDAP.php]: please use tabs, not spaces

**Review by blizzz — APPROVED** [review]
Makes sense, thank you 👍 

Could you sign off your commit (``git commit -s``) so that the checker also passes?

**Comment by blizzz** [comment]
need a second reviewer @nextcloud/ldap

**Comment by blizzz** [comment]
Fix for #456

**Review by rullzer — APPROVED** [review]

**Pull request merged**